### DISCUSSION TITLE
Loading depot vs stations?

### DISCUSSION THREAD

**Discussion by freddepettersson** [tellurianinteractive/Tellurian.Trains.ModulesRegistryApp#41] [Q&A] [answered]
Hi
How shall a loading depot be setup in the system ? in case there are no person transport to this area , only cargo.
Stations - is this name general for all traffic areas that need to be added for being able to make a timeschedule possible  in the traffic game ?
Just asking to make something more clear or understandable for the user

/F

**Comment by fjallemark** [comment, marked as answer]
It will only be possible to register train times at 'stations'. The simplest 'station' is a junction with one switch. If a freight station has scheduled operation you need to define the tracks that can be scheduled. If there are no platforms for passenger, simply don't specify a platform length. All loading tracks or areas are normally not scheduled and does not need to be specified as track. You can add all cargo customers and specify a track number or loading area designation for each customer, and you can then overide rack number or loading area designation for each cargo flow.
